README.md

```diff
@@ -6,6 +6,12 @@
 
 This repo contains the React/Gatsby application that powers [the EDB Docs website](https://www.enterprisedb.com/docs/). The site pulls [Markdown](https://www.markdownguide.org/) content from several repos in a process called "sourcing", and then renders it all into high-performance markup. You can install the application on your local computer for easy editing, viewing, and eventually publishing to the GitHub repo.
 
+## Please remove and re-clone your local repositories after August 6, 2021
```

Comment:
Sooner we do this the better. I'm not sure waiting a week will do any good, so I suggest ripping the band aid off this weekend.

Reply:
That works. I would suggest that you create or sync a fork right before you blow the old repo away.

Also, we'll have to give notice to folks working on the docs repo that they'll need to be sure to push their changes up, or stash them away at the end of Friday.

Reply:
Creating a new fork makes sense. It's not as if we're trying to hide API keys or something. :-)

Tomorrow I'll send out something internally to people who have touched the repo. Not sure what to do about external contributors or even how many we might have.